src/ilasm/CMakeLists.txt

```diff
@@ -24,6 +24,12 @@ set(ILASM_SOURCES
   assembler.cpp
   prebuilt/asmparse.c
 )
+if(WIN32)
+  set(ILASM_RESOURCES Native.rc)
+endif(WIN32)
+
+add_definitions(-DFX_VER_INTERNALNAME_STR=ilasm.exe)
```

Comment:
Given the ".exe" part, should this ideally be under a if(WIN32) even though it may not currently have any effect outside that setting anyway?